common/test/acceptance/tests/test_cohorted_courseware.py

```diff
@@ -48,6 +54,26 @@ def setUp(self, is_staff=True):
             self.browser, username=self.cohort_b_student_username, email=self.cohort_b_student_email, no_login=True
         ).visit()
 
+        # Create a Verified Student
+        self.cohort_verified_student_username = "cohort_verified_student"
+        self.cohort_verified_student_email = "cohort_verified_student@example.com"
+        StudioAutoAuthPage(
+            self.browser,
+            username=self.cohort_verified_student_username,
+            email=self.cohort_verified_student_email,
+            no_login=True
+        ).visit()
+
+        # Create audit student
+        self.cohort_audit_student_username = "cohort_audit_student"
```

Comment:
@staubina Thinking about this overnight (OK, I promise I didn't give it TOO much thought overnight), another alternative that I think is acceptable is to add comments in the methods that check which components are visible to the "cohort" students. Have the comments state that the learner is auto-enrolled in the Honor track, and therefore can't see the Audit (or Verified) content.

Although it doesn't mimic "reality", at least it helps explain the expected/tested behavior for the next person reading the code.